test/cpp/jit/test_save_load.cpp

```diff
@@ -13,6 +15,20 @@
 namespace torch {
 namespace jit {
 
+namespace {
+
+static Module roundtrip(const Module& m) {
+  ExtraFilesMap files;
+  std::vector<IValue> constants;
+  jitModuleToPythonCodeAndConstants(m, &files, &constants);
+  CompilationOptions options;
+  mobile::Module mobilem = jitModuleToMobile(m, options);
```

Comment:
Could we also save the extra files to mobile module to have a more complete roundtrip?

Reply:
extra files is orthogonal. 
The full roundtrip is
jit::Module -> (ivalue, source, constant) -> flatbuffer -> (ivalue, source, constant) -> jit::Module.
This test only tests first first and last step:
jit::Module -> (ivalue, source, constant)->jit::Module.

A test with extrafiles would be better aimed at testing torch::jit::load when it handles flatbuffer.